tests/stress-tests.lisp

```diff
@@ -28,9 +28,10 @@
   (write-char #\]) (finish-output)
   nil)
 
-#+unix
 (deftest test-posix-shared-memory-allocation/deallocation ()
   "Test allocation and deallocation of large blocks of POSIX shared memory. This test may result in total system failure."
+  #+windows (skip)
+  #-windows
   (run-unless-environment-has "DISABLE_SHARED_MEMORY_QVM_TESTS"
```

Comment:
WHAT about changing the `run-unless` macro to skip? This isn't a demand, but a question, but the question blocks merging this PR.